src/data/quot.lean

```diff
@@ -133,6 +141,28 @@ protected lemma induction_on₃
   (h : ∀ a b c, δ (quot.mk r a) (quot.mk s b) (quot.mk t c)) : δ q₁ q₂ q₃ :=
 quot.ind (λ a₁, quot.ind (λ a₂, quot.ind (λ a₃, h a₁ a₂ a₃) q₃) q₂) q₁
 
+instance (r : α → α → Prop) (f : α → Prop) (h : ∀ a b, r a b → f a = f b)
```

Comment:
Can you check what the generated instance name is here?

Reply:
Looks just fine.